src/components/Footer.jsx

```diff
@@ -20,15 +20,18 @@ export default function Footer() {
                 
                 </div>
                 <div className="footer-links">
-                    {/* <p className="footer-title">Links</p>
-                    <a href="" className="website-link">www.website.com</a>
-                    <a href="" className="website-link">www.website.com</a> */}
+                    <p className="footer-title">Links</p><br />
+                    <h3>Developers: Dashona Holt, Lucile Jones, Natalie Taylor, Benjamin Vredevelt</h3> <br />

```

Comment:
Change verbiage from `Developers: Dashona Holt, Lucile Jones, Natalie Taylor, Benjamin Vredevelt`

to `Made By:`

Reply:
On it!